Mutagen.Bethesda.Analyzers.Api/AnalyzerOptions.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Mutagen.Bethesda.Analyzers.Config.Topic;
+using Mutagen.Bethesda.Analyzers.SDK.Topics;
+
+namespace Mutagen.Bethesda.Analyzers.Api;
+
+public class AnalyzerOptions : IMinimumSeverityConfiguration
+{
+    public Dictionary<TopicId, Severity> SeverityOverrides { get; set; } = new();
```

Comment:
Id rather this specify a TopicConfig that the user can pass in.  That way, if we add functionality to TopicConfig to help easier specification of things, the user can use it, rather than just dealing with a naked dictionary.